lib/pbench/cli/server/shell.py

```diff
@@ -1,8 +1,13 @@
 #!/bin/env python3
 
 from pbench.server.api import create_app
+import subprocess
 
 
 def main():
     app = create_app()
-    app.run(debug=True, port=app.config["PORT"])
+    port = app.config["PORT"]
+    host = app.config["BIND_HOST"]
+    workers = app.config["WORKERS"]
+    subprocess.run(["gunicorn", "--workers", str(workers), "--pid", "/opt/pbench-server/gunicorn.pid",
```

Comment:
This apparently would be reformatted via `black`, as the lint job in Travis CI is failing.